test/integration/generated/iotspaces/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "module": "es6",
+    "moduleResolution": "node",
+    "strict": true,
+    "target": "es6",
```

Comment:
No need to be fixed in this PR but I think we can target ES2017 everywhere now to be consistent with Track 2 packages in the mono repo.

Reply:
Will do it in a seperate PR